Microsoft.DotNet.Interactive.Rendering/Microsoft.DotNet.Interactive.Rendering.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        $CommonMetadataElements$
+        <language>en-US</language>
+         <dependencies>
+            <group targetFramework=".NETStandard2.0">
+                <!-- Dependencies of Microsoft.Dotnet.Interactive.Rendering -->
+                <dependency id="Microsoft.AspNetCore.Html.Abstractions" version="$MicrosoftAspNetCoreHtmlAbstractionsVersion$" />
+                <dependency id="microsoft.csharp" version="$microsoftcsharpVersion$" />
+               
+                <!-- Dependencies of MLS.Agent.Tools -->
+                <dependency id="Clockwise" version="$ClockwiseVersion$" />
```

Comment:
This should not have a Clockwise dependency.

Reply:
Moved Dotnet and CommandLine classes upto Workspace server hence removing the clockwise dependency here